exporters/powerdns/powerdns.json.tmpl

```diff
@@ -1,8 +1,21 @@
 {
-    "arguments": {
```

Comment:
is wired that this was working in the first place, i guess the agent also is using that but in an unsupported way or something like that